### PR TITLE
=doc #22373 Change Future to CompletionStage in Java documentation

### LINKED ISSUE #22373: [Docs] Java documentation regarding 'ask' referencing Scala API instead of Java API
The documentation on the Java ['Actors'](http://doc.akka.io/docs/akka/2.4/java/lambda-actors.html#Ask__Send-And-Receive-Future) page uses the Scala version of 'ask' instead of the one in PatternsCS.

### PR DESCRIPTION
**Pull request by talpr**
Resolves #22373 

It turns out the updated documentation already mentioned PatternsCS.ask, but the sample and main explanation still used Future, so I changed the sample to use CSs and changed the documentation accordingly. I also changed the sample for gracefulStop, while I was at it...

### CODE REVIEW

**Comment by hepin1989** [comment]
A big thanks.

**Review by johanandren — COMMENTED** [review]
LGTM

**Review by johanandren — APPROVED** [review]

File: akka-docs/.history [hunk 1]
```diff
@@ -1 +0,0 @@
-exit
```
Comment (patriknw) [inline akka-docs/.history]: what kind of file is this? should it instead be removed and added to .gitignore

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Comment by patriknw** [comment]
OK TO TEST

File: akka-docs/.history (hunk 1 above)
Comment (ktoso) [inline akka-docs/.history]: hm that's a bash thing, it should be deleted entirely. I wonder how it ended up here

Comment (talpr) [inline akka-docs/.history]: @ktoso @patriknw I'm not sure, it was there in the commit I originally based my PR on so I removed it, and the file is already ignored. I'm not sure what happened during the rebase, but I don't see the file in my repo...
I could try to amend it if it's still there.

**Comment by talpr** [comment]
Is the test failure related to this commit? Is there anything I should fix?

**Comment by patriknw** [comment]
@talp-traiana I think something similar was fixed in https://github.com/akka/akka/pull/22472
Please rebase.

**Comment by talpr** [comment]
@patriknw Hmm, looks like #22472 includes the changes I've made here already, so I'm closing this PR.

**Pull request closed without merging**